clients/privacy-center/components/consent/SaveCancel.tsx

```diff
@@ -1,31 +1,38 @@
 import { Button, Stack, StackProps } from "@fidesui/react";
+import useI18n from "~/common/hooks/useI18n";
 
 const SaveCancel = ({
   onSave,
   onCancel,
-  saveLabel = "Save",
+  saveLabel,
+  cancelLabel,
   ...props
 }: {
   onSave: () => void;
   onCancel: () => void;
   saveLabel?: string;
-} & StackProps) => (
-  <Stack direction="row" justifyContent="flex-start" width="full" {...props}>
-    <Button size="sm" variant="outline" onClick={onCancel}>
-      Cancel
-    </Button>
-    <Button
-      bg="primary.800"
-      _hover={{ bg: "primary.400" }}
-      _active={{ bg: "primary.500" }}
-      colorScheme="primary"
-      size="sm"
-      onClick={onSave}
-      data-testid="save-btn"
-    >
-      {saveLabel}
-    </Button>
-  </Stack>
-);
+  cancelLabel?: string;
+} & StackProps) => {
+  const { i18n } = useI18n();
+
+  return (
+    <Stack direction="row" justifyContent="flex-start" width="full" {...props}>
+      <Button size="sm" variant="outline" onClick={onCancel}>
+        {cancelLabel || "←"}
+      </Button>
```

Comment:
The default here is going to be problematic for accessibility; I'm assuming this was meant to avoid translation. We need to translate this one way or another and include it as an `aria-label` if we're just using an icon. Maybe best just to include the translated label similar to the save button?

Reply:
Yes, you're it, it's not accesible. I am adding an aria-label with hardcoded value of "Cancel". We reviewed it during a refinement call and agreed to just use the icon specifically to not have to add new translations strings.